services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/_meta.py

```diff
@@ -5,7 +5,7 @@
 from pydantic import BaseModel, HttpUrl
 
 from .._meta import API_VERSION, PROJECT_NAME
-from ._dependencies import get_reverse_url_mapper
+from .dependencies import get_reverse_url_mapper
```

Comment:
Q? `dependencies` submodule is not internal to `api` package?

Reply:
```
async def list_containers(
    user_id: UserID,
    product_name: ProductName,
    page_params: Annotated[LimitOffsetParamsWithDefault, Depends()],
    resource_tacker_repo: ResourceTrackerRepository = Depends(
        get_repository(ResourceTrackerRepository)
    ),
```